src/app/k-tools/k-tool-form/k-tool-form.component.html

```diff
@@ -0,0 +1,42 @@
+<h2>{{ kTool ? "Edit" : "New" }} Tool</h2>
```

Comment:
Would you mind throwing this in a mat-card (title, content, footer)? I set up a _modal.scss file to style modal forms in another ticket. But at least this way they will all be mat-cards

Reply:
If I wrap the form in a mat-card, it will look strange when displayed using the dialog since the dialog already looks like a card... I think we should keep the form components as simple as possible so we can put it anywhere. I could do mat-card-title and all of that within the form which should still work without mat-card but allow us to wrap the form in a mat-card if thats what we want? Or we could define our own non-material classes in a similar way for standard styles?

Reply:
I'd like for all the forms to be uniform as possible.  Having everything contained in a mat-card (header, footer, etc) will make is easier to accomplish this and easier to style without having to add a bunch of our own custom classes. 

_forms.scss will handle our own edits to material's styles for forms in mat-cards with the assumption it will be displayed not in a modal (if/when we want to do that).
_modals.scss will handle forms styling in dialogs. I have some css already in to address the extra padding.